core/tests/webdriverio_utils/ReleaseCoordinatorPage.js

```diff
@@ -156,70 +140,14 @@ var ReleaseCoordinatorPage = function() {
   };
 
   // This function is meant to be used to enable a feature gated behind
-  // a feature flag in test mode, which is the server environment the E2E
-  // tests are run in.
-  this.enableFeatureForTest = async function(featureElement) {
-    await action.click(
-      'Add feature rule button',
-      featureElement.$(addFeatureRuleButtonLocator)
-    );
-
-    await waitFor.visibilityOf(
-      featureElement.$(valueSelectorLocator),
-      'Value Selector takes too long to appear'
-    );
-    await (featureElement.$(valueSelectorLocator)).selectByVisibleText(
-      'Enabled');
-
-    await action.click(
-      'Add condition button',
-      featureElement
-        .$(addConditionButtonLocator)
-    );
-
-    await waitFor.visibilityOf(
-      featureElement.$(serverModeSelectorLocator),
-      'Value Selector takes too long to appear'
-    );
-    await (featureElement.$(serverModeSelectorLocator)).selectByVisibleText(
-      'test');
-
-    await this.saveChangeOfFeature(featureElement);
-  };
-
-  // This function is meant to be used to enable a feature gated behind
-  // a feature flag in prod mode, which is the server environment the E2E
-  // tests are run in.
+  // a feature flag.
   this.enableFeature = async function(featureElement) {
-    await this.removeAllRulesOfFeature(featureElement);
-
-    await action.click(
-      'Add feature rule button',
-      featureElement
-        .$(addFeatureRuleButtonLocator)
-    );
     await waitFor.visibilityOf(
       featureElement.$(valueSelectorLocator),
       'Value Selector takes too long to appear'
```

Comment:
Maybe you should give more detailed info in this string so if a failure occurs here it's clearer what exact element caused the problem.

Reply:
Updated